[Node.js] Style guide fixes for TDBX vale rules (#842)
mongodb/docs-node: source/usage-examples/bulkWrite.txt

  the `bulkWrite API documentation <{+api+}/classes/Collection.html#bulkWrite>`__.

- ``options``: *optional* settings that affect the execution
  of the operation, such as whether the write operations executes in
  sequential order and the write concern.

  By default, MongoDB executes bulk write operations one-by-one in the
  specified order (serially). During an ordered bulk write, if
  an error occurs during the processing of an operation, MongoDB returns
  without processing the remaining operations in the list. In contrast,
  when ``ordered`` is ``false``, MongoDB continues to process remaining
  write operations in the list. Unordered operations are theoretically faster

>   since MongoDB can execute them in parallel, but will only be used if
>   the writes do not depend on order.

Reviewer: ```suggestion
  since MongoDB can execute them in parallel, but use them only if
  the writes do not depend on order.
```

         :language: typescript
         :linenos:

> When you run the preceding example, you will see the following output:

Reviewer: ```suggestion
Running the preceding example results in the following output:
```
